apisix/cli/ngx_tpl.lua

```diff
@@ -765,6 +765,11 @@ http {
             }
             {% end %}
 
+            {% if proxy_mirror_timeout then %}
+                {% for k, v in pairs(proxy_mirror_timeout) do %}
+            proxy_{* k *}_timeout {* v *}ms;
```

Comment:
This way we may pollute the nginx template, we may write some bad directive like: "proxy_foo_timeout".

Reply:
updated